themes/V3/5ePHB/style.less

```diff
@@ -677,6 +677,135 @@ body {
 		margin-top : 0.2cm;
 	}
 }
+//*****************************
+// *      COVER PAGE
+// *****************************/
+
+.page {
+	&:has(.coverPage):after {
+		display: none;
+	}
+	.coverPage {
```

Comment:
I wonder if it'd be a good idea to include the class name "default", and set the css to target `.page .default.coverPage`.  This way someone could still use the class name `coverPage` themselves and completely ignore the default styling without having to unset/override a bunch of existing properties.....and then if they want to have the default styling they just add `default`.  

I don't know if that makes much sense, though.  Especially with `@layers`, maybe that makes this a moot point.

Reply:
That could be a thing, but it's the same as users using `hardCover` or `myCover` instead of `coverPage`, which they will, because they won't know they can use `coverpage` without importing styles.